### PR TITLE
Homework11: Alina Shautsova

### PR DESCRIPTION
**Pull request by AlinaShautsova**

### CODE REVIEW

File: homework11_bank_deposit.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        the deposit term in years, the annual interest rate."""
+        months = self.deposit_term * 12
+        monthly_rate = self.annual_rate / 12 / 100
+        final_amount1 = self.sum_of_deposit
+
+        for _ in range(months):
+            final_amount1 += final_amount1 * monthly_rate
+        return final_amount1
+
+    def get_sum_of_deposit(self):
```
Comment (Stml89) [inline homework11_bank_deposit.py]: `get_deposit_amount`

File: homework11_library.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    def __init__(self, title, author, number_of_pages, isbn):
+        """Function initializes a new instance of the Book class. """
+        self.title = title
+        self.author = author
+        self.number_of_pages = number_of_pages
+        self.isbn = isbn
+        self.reserved_by = ""
+        self.took_by = ""
+
+    def get_details(self):
```
Comment (Stml89) [inline homework11_library.py:22]: Use `__str__`

File: homework11_library.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+
+class User:
+    """This class include 4 functions for initializes, for reserve, take and
+     return the book."""
+    def __init__(self, name):
+        """Function initializes a new instance of the User class. """
+        self.name = name
+
+    def reserve_the_book(self, book):
```
Comment (Stml89) [inline homework11_library.py]:
```suggestion
    def reserve_book(self, book):
```

**Review by Stml89 — APPROVED** [review]
Fix small comments and you can merge PR

File: homework11_bank_deposit.py (hunk 1 above)
Comment (AlinaShautsova) [reply homework11_bank_deposit.py]: Done

File: homework11_library.py (hunk 2 above)
Comment (AlinaShautsova) [reply homework11_library.py:22]: Done

File: homework11_library.py (hunk 3 above)
Comment (AlinaShautsova) [reply homework11_library.py]: Done

**Pull request merged**